Carbon/navigation/screens/Main/MainContainer.js

```diff
@@ -73,7 +73,7 @@ const ProgressStack = ({ navigation }) => {
     return (
         <Stack.Navigator>
             <Stack.Screen
-                name={' '}
+                name={'Your Progress'}
```

Comment:
Note, I've set up the monthly progress as part of the daily log, feel free to use the built in function to get that data.
It would be calling GetData() which returns an array of arrays with 4 distinct arrays in it representing day, yesterday, week, and month. If you access 2dArray[3] you will get all the values averaged out for the past month. If that would be helpful.